docs/v1/api_auth_methods.md

```diff
@@ -0,0 +1,63 @@
+  # Auth method API Calls
+
+  
+  ## [GET] Manage Services - ListOneAuthMethod
+  
+  ### Request
+  
+  `GET/v1/services/{service}/hosts/{host}/authM`
+  
+   If the request is successful, the response contains information for the requested auth method.
+   
+   Success Response
+   
+   `200 OK`
+   
+```json
+{
+    "_id": "5af36ae263927f878860c3f0",
```

Comment:
Choose a uuid instead of exposing mongo object id. Also in mock store the schema doesn't contain an exposed id.

Reply:
i have in mind, to remove it in a later pr